src/pages/home/HeaderView.js

```diff
@@ -45,28 +46,33 @@ const propTypes = {
         guideCalendarLink: PropTypes.string,
     }),
 
+    parentReportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)),
+
     ...windowDimensionsPropTypes,
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     personalDetails: {},
+    parentReportActions: {},
     report: null,
     account: {
         guideCalendarLink: null,
     },
 };
 
 const HeaderView = (props) => {
+    // TO DO: Replace with subscribing to specific action rather than all.
```

Comment:
Generally for todos we ask people to link a GH issue so that its progress can be tracked and not forgotten. Can you add that here.